doc/userdoc/nest_server.rst

```diff
@@ -0,0 +1,649 @@
+NEST Server
+===========
+
+.. contents:: On this page, you'll find
+   :local:
+   :depth: 1
+
+What is NEST Server?
+--------------------
+
+NEST Server enables users to interact with the NEST simulation engine
+via a RESTful API. Using this approach, you can perform the same basic
+operations as with :doc:`PyNEST <ref_material/pynest_apis>`, but
+instead of doing so by directly importing the ``nest`` module, all
+commands as well as their arguments and result data are channeled
+through HTTP requests and responses transmitted over a TCP/IP
```

Comment:
yes I agree, I misinterpreted the sentence. 
How about this:

"Using this approach, you can perform the same basic operations as with :doc:`PyNEST <ref_material/pynest_apis>`, but instead of directly importing the nest module, all commands, including their arguments and result data, are channeled through HTTP requests and responses over a TCP/IP connection."

Reply:
I like how adding more commas seems to be your generic answer to everything :-)
I also like your new version and have added a suggestion those lines below.